### DISCUSSION TITLE
Ask Anything

### DISCUSSION THREAD

**Discussion by nicocodemance** [nicocodemance/country-cities#4] [Q&A] [answered]
Please ask anything if you have any questions

**Comment by cocoachcom** [comment, marked as answer]
What it means the flag "🇺🇸"?

**Reply by nicocodemance** [reply]
That is the unicode from country flags

Please refer [this link](https://www.alt-codes.net/flags)

**Reply by cocoachcom** [reply]
Aha, yes. Thank you

**Reply by purecodecoach** [reply]
Very useful!!!
